packages/manager/apps/dedicated/client/app/account/user/ip/restriction/user-ip-restriction.service.js

```diff
@@ -8,10 +8,20 @@ export default /* @ngInject */ function UserAccountIpRestrictionsService(
 
   this.getList = function getList() {
     return $http
-      .get('/sws/me-access-restriction-ip', {
-        serviceType: 'aapi',
+      .get('/me/accessRestriction/ip', {
+        headers: {
+          Pragma: 'no-cache',
+          'X-Pagination-Mode': 'CachedObjectList-Pages',
+          'X-Pagination-Size': '5000',
+        },
       })
-      .then(getSuccessDataOrReject);
+      .then(getSuccessDataOrReject)
+      .then((data) => {
+        return data.map((ipRestiction) => {
+          ipRestiction.rule = ipRestiction.rule.toUpperCase(); // eslint-disable-line no-param-reassign
+          return ipRestiction;
+        });
```

Comment:
To avoid disabling eslint rule
```suggestion
        return data.map((ipRestiction) => ({
          ...ipRestriction,
          rule: ipRestiction.rule.toUpperCase(),
        }));
```

Reply:
Rebasing on `develop` branch in order to apply suggestion.

Reply:
Also fixing typo (`s/ipRestiction/ipRestriction/`)